model-optimizer/mo/utils/class_registration.py

```diff
@@ -82,11 +82,13 @@ def _update(cls, registered_list: list, registered_dict: dict, key: str, enabled
 
     for c in cls.__subclasses__():
         # Force enabling operations
-        if hasattr(c, 'id') and c.id in enabled_transforms:
+        if hasattr(c, 'id') and c.id in enabled_transforms or \
```

Comment:
Please, update the `./docs//MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md` file with information about new way of enabling/disabling transformation.

Reply:
Done.